### DISCUSSION TITLE
Nothing happens when I click process application.

### DISCUSSION THREAD

**Discussion by tacticalNukes** [community/community#87429] [GitHub Education] [answered]
### Hi everyone,

Hello, when I apply for the student developer back and i've put in an image and click process application nothing happens. The button goes unclickable and I get nothing else. If I reload the page I can apply again like I haven't applied. I get no email or anything about me having applied for it. I am wondering is this a normal thing or am I doing something wrong? Also get error in my chrome console
`Failed to load resource: the server responded with a status of 413 ()`
`Uncaught TypeError: Cannot read properties of undefined (reading 'redirect_url')`

**Comment by afeuerstein** [comment]
Seems to be the same problem as here: https://github.com/orgs/community/discussions/88401#discussioncomment-8148320
(#88401)

**Comment by tacticalNukes** [comment]
This solved it, thank you very much!

**Reply by mishmanners** [reply]
Glad you got it working. I'll close out this Discussion for now that you have your question resolved.

**Reply by mishmanners** [reply, marked as answer]
If anyone else finds this and experiences the same issue, please try:
- Refreshing the page or clear your browser's cache and cookies.
- Check for browser extensions that could be interfering with the JavaScript code and disable them
- Try a different browser
